pkg/package-server/client/clientset/versioned/typed/packagemanifest/v1alpha1/packagemanifest.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by client-gen. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	v1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/packagemanifest/v1alpha1"
+	scheme "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/client/clientset/versioned/scheme"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	watch "k8s.io/apimachinery/pkg/watch"
+	rest "k8s.io/client-go/rest"
+)
+
+// PackageManifestsGetter has a method to return a PackageManifestInterface.
+// A group's client should implement this interface.
+type PackageManifestsGetter interface {
+	PackageManifests(namespace string) PackageManifestInterface
+}
+
+// PackageManifestInterface has methods to work with PackageManifest resources.
+type PackageManifestInterface interface {
+	Create(*v1alpha1.PackageManifest) (*v1alpha1.PackageManifest, error)
+	Update(*v1alpha1.PackageManifest) (*v1alpha1.PackageManifest, error)
+	UpdateStatus(*v1alpha1.PackageManifest) (*v1alpha1.PackageManifest, error)
+	Delete(name string, options *v1.DeleteOptions) error
+	DeleteCollection(options *v1.DeleteOptions, listOptions v1.ListOptions) error
+	Get(name string, options v1.GetOptions) (*v1alpha1.PackageManifest, error)
+	List(opts v1.ListOptions) (*v1alpha1.PackageManifestList, error)
+	Watch(opts v1.ListOptions) (watch.Interface, error)
+	Patch(name string, pt types.PatchType, data []byte, subresources ...string) (result *v1alpha1.PackageManifest, err error)
+	PackageManifestExpansion
+}
+
+// packageManifests implements PackageManifestInterface
+type packageManifests struct {
+	client rest.Interface
+	ns     string
+}
+
+// newPackageManifests returns a PackageManifests
+func newPackageManifests(c *PackagemanifestV1alpha1Client, namespace string) *packageManifests {
+	return &packageManifests{
+		client: c.RESTClient(),
+		ns:     namespace,
+	}
+}
+
+// Get takes name of the packageManifest, and returns the corresponding packageManifest object, and an error if there is any.
+func (c *packageManifests) Get(name string, options v1.GetOptions) (result *v1alpha1.PackageManifest, err error) {
```

Comment:
Is there a way to tell the Client codegen to only generate get/list/watch?

Reply:
Yes - I'm testing that out now.